doc/level1.rst

```diff
@@ -726,19 +726,33 @@ keys instead.  The MUA MUST be capable of assembling these keys into
 an OpenPGP certificate (:rfc:`RFC 4880 "Transferable Public
 Key"<4880#section-11.1>`) that indicates these capabilities.
 
+Secret key protection at rest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 The secret key material should be protected from access by other
 applications or co-tenants of the device at least as well as the
 passwords the MUA retains for the user's IMAP or SMTP accounts.
 
-Secret key protection at rest
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The MUA SHOULD NOT protect the secret key with a password. All
-encrypted outgoing messages MUST be signed, which would require the
-user to enter the password for both reading and sending mail. This
-introduces too much friction to become part of a routine daily workflow.
-Protection of the user's keys at rest and other files is achieved more
-easily and securely with full-disk encryption.
+The MUA MAY protect the secret key (and other sensitive data it has
+access to) with a password, but it SHOULD NOT require the user to
+enter the password each time they send or receive a mail.  All
+encrypted outgoing messages MUST be signed, so if the user has to
+enter the password for both reading and sending mail, this introduces
+too much friction to become part of a routine daily workflow.
+
+Note that password protection of the secret key carries with it a risk
+that the user might forget their password, which might result in
+catastrophic data loss.  Unlike IMAP or SMTP credentials (which can be
+reset by the server operator given some sort of out-of-band
+confirmation), there is no recovery workflow possible for the loss of
+a password protecting a secret key.  An MUA that chooses to offer
+password protection of the secret key (or other sensitive data) SHOULD
+support usable and secure backup/recovery workflows for the protected
```

Comment:
Backup/recovery workflows for lost passphrases? How is this possible without making passphrases totally useless?

Reply:
Maybe that's an exercise for the reader?  :)  Note that the backup/recovery workflows aren't for the password itself, but for the material protected by the password.

Here are some examples:

 * secret-sharing split among trusted contacts
 * [the `paperkey` package](http://www.jabberwocky.com/software/paperkey/)
 * a hand-written Setup Code, stored offline in a safe, combined with the associated Autocrypt Setup Message

I think describing concrete examples of backup/recovery is out of scope for Level 1, but i do think we should mention it for implementers so that they don't accidentally burn their users.

Reply:
then I have no objections :)